lib/gcloud/resource_manager.rb

```diff
@@ -84,9 +84,13 @@ def self.resource_manager keyfile = nil, options = {}
   #
   # == Accessing the Service
   #
-  # The Resource Manager API *cannot* be enabled in the developer console, and
-  # can only be enabled by your account manager or a member of the Google Cloud
-  # sales team.
+  # Currently, the full functionality of the Resource Manager API is available
+  # only to whitelisted users. (Contact your account manager or a member of the
+  # Google Cloud sales team if you are interested in access.) The read-only methods
+  # {projects.list}[https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/list] and
```

Comment:
What do you think about referencing the gcloud methods instead of the API methods here? So linking to `Manager#projects` here instead of `projects.list`?